addons/hr/models/hr.py

```diff
@@ -388,6 +388,12 @@ class Department(models.Model):
     note = fields.Text('Note')
     color = fields.Integer('Color Index')
 
+    @api.multi
+    def name_get(self):
+        if self.env.context.get('in_search_panel', False):
```

Comment:
`get()` returns a falsy value when the key is not found. Idiomatic code gives
```suggestion
        if self.env.context.get('in_search_panel'):
```